frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java

```diff
@@ -30,13 +32,7 @@ public static String getSampleMacheteFileContents() {
 
   @UIThreadUnsafe
   public static List<String> getBranchNamesForPsiMacheteFile(PsiFile psiFile) {
-    val project = psiFile.getProject();
-
-    val gitRepository = List.ofAll(GitRepositoryManager.getInstance(project).getRepositories())
-        .find(repository -> {
-          val macheteFile = repository.getMacheteFile();
-          return macheteFile != null && macheteFile.equals(psiFile.getVirtualFile());
-        });
```

Comment:
extracting to `findGitRepositoryForPsiMacheteFile` method as it would be useful in the `CreateBranchQuickFix` class in another file